### DISCUSSION TITLE
[HTTP] How to configure proxypass

### DISCUSSION THREAD

**Discussion by bzajacc** [roadrunner-server/roadrunner#1215] [Q&A] [answered]
Hi,
how can i configure proxy_pass in RR? Now in my nginx.conf I have something like this

```shell
location /foo/bar/ {
    proxy_set_header Host $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_pass http://some.other.url/foo/bar/new;
}
```

**Comment by rustatian** [comment, marked as answer]
Hey @bzajacc 👋🏻 , nice to meet you ⚡ 

Yeah, you may use a `proxy_pass` to the `127.0.0.1:[http.address(rr.yaml)]` address or `fastcgi_pass`, because RR also supports a `fastcgi`: https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml#L812
